### DISCUSSION TITLE
Plot customization in interactive mode

### DISCUSSION THREAD

**Discussion by matteogabba** [scikit-hep/iminuit#1007] [Q&A]
I'd like to customize the plot appearance in interactive mode.
More specifically, I want to fit a time decaying signal with a multiexponential model and being able to plot on logx-scale would really help.
I tried to pass kwargs to m.interactive() as suggested in the docstrings but I got an error message.
Some help would be helpful.
Thanks.

**Comment by HDembinski** [comment]
You can plug in your own plotting code with the `plot` keyword.
